enterprise/backend/src/metabase_enterprise/serialization/api.clj

```diff
@@ -129,7 +140,8 @@
    field_values     [:maybe ms/BooleanValue]
    database_secrets [:maybe ms/BooleanValue]}
   (api/check-superuser)
-  (let [opts               {:targets                  (mapv #(vector "Collection" %)
+  (let [start              (System/currentTimeMillis)
```

Comment:
nitpick: This method uses the system clock and is not guaranteed monotonic, it's preferable to use `System/nanoTime` whenever measuring elapsed time, as it avoids the overhead of a system call and precisely measures elapsed time since the process started.

Reply:
eh, I don't think precision is necessary here - serialization is usually in tens or hundreds of seconds

Reply:
The issue is not precision, it is accuracy. 

I'm not blocking on this, but "in my book" it's best to strictly use nanoTime for timing things - it saves you thinking about whether you need to worry about negative numbers or making an inner loop slower.

Reply:
I found a good quote on StackOverflow to summarize:

> The purpose of nanoTime is to measure elapsed time, and the purpose of currentTimeMillis is to measure wall-clock time. You can't use the one for the other's purpose.